app/collections/settings.jsx

```diff
@@ -84,6 +89,37 @@ export default class CollectionSettings extends React.Component {
       });
   }
 
+  handleDescriptionInputChange() {
+    const property = 'description';
+    const setting = this.state.setting;
+    setting[property] = true;
```

Comment:
Is this a copy and paste from the `handleToggle` event handler? This handler is dealing with text, not boolean values of a checkbox or radio button input type, so setting this to true doesn't make sense!